package.json

```diff
@@ -71,7 +71,6 @@
   "scripts": {
     "clean": "rimraf logs/*",
     "lint": "jscs .",
-    "pretest": "npm run clean && npm run lint",
```

Comment:
You removed the pretest script, which is there to run the linting before the tests run.